### ISSUE TITLE
Monoprice Recessed Door/Window Sensor

### ISSUE THREAD

**Issue by kjohns3** [openhab/org.openhab.ui.habmin#243]
@cdjackson can you help add the Monoprice Recessed Door/Window Sensor?  I've attached the node.xml file that was generated by openhab2.

[node28.txt](https://github.com/openhab/org.openhab.ui.habmin/files/559420/node28.txt)
##

**Comment by cdjackson** [comment]
Can you provide a link to the manual please?

**Comment by kjohns3** [comment]
I was unable to find a link to a PDF on the monoprice website for this particular device.  I've attached a scanned version of the manual that came with the device.  If there's any part of the manual that's difficult to read, let me know and I'll be happy to recreate the data in text form from the manual.

Thanks, Ken

[Monoprice 15268 Manual.pdf](https://github.com/openhab/org.openhab.ui.habmin/files/559609/Monoprice.15268.Manual.pdf)

**Comment by xsnrg** [comment]
This device already exists in the database for OpenHAB2.  You didn't mention when you last updated OpenHAB2, or I guess even if you were using OpenHAB2.  Here is the link to the entry.  I have one of these devices working well.

http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/420

**Comment by kjohns3** [comment]
@xsnrg Thanks for the link!

Well, I upgraded from openhab2 beta3 to beta4 today.  Actually, I ended up doing a clean install.  When I look at the "Supported Thing Types" from the Bindings view, I don't see ZD2105 listed.

The current version of HABmin is version: 0.1.7, 2016-09-14T17:38:17

The device you linked to was last updated on 2016-10-09 18:28:04, which means it was added after beta4 was released.  Is there any way to update the zwave database without using the snapshot release of openhab2?

Thanks,
Ken
